app/internal/page/donor/check_your_lpa.go

```diff
@@ -34,6 +34,12 @@ func CheckYourLpa(tmpl template.Template, donorStore DonorStore, shareCodeSender
 			data.Errors = data.Form.Validate()
 
 			if data.Errors.None() {
+				redirect := page.Paths.LpaDetailsSaved.Format(lpa.ID)
```

Comment:
this is an aside, but should we be auto-checking the `Form.CheckedAndHappy` with your previous answer now that it triggers new stuff?

Reply:
That's a good point - it feels like this should always require a reconfirmation especially when we come to implement Beth's new designs. I'll drop in the text message tickets I'm working on now.